colour-changing-grid/style.css

```diff
@@ -1 +1,14 @@
-/* Colour changing grid CSS here */
\ No newline at end of file
+/* Colour changing grid CSS here */
+div {

```

Comment:
I understand that this is a simple exercise, so the markup is also simple, but it's not good practice to use CSS rules on `div` itself. A class would be better here, especially if it was named semantically.

Again, it's not necessary for this, I just wanted to point it out for future more complicated stuff.